features/hangouts/edit_hangout_url.feature

```diff
@@ -30,4 +30,12 @@ Feature: Manual Edit of Hangout URL
     And I manually set a hangout link for event "Repeat Scrum"
     Then "Repeat Scrum" shows live for that hangout link for the event duration
     And "Repeat Scrum" is not live the following day
-    And there should be exactly 3 hangouts
\ No newline at end of file
+    And there should be exactly 3 hangouts
+
+  Scenario: Event doesn't go live before Hangout URL is updated
+    Given the date is "2014 Feb 5th 6:00am"
+    And I manually set a hangout link for event "Repeat Scrum"
+    Given the date is "2014 Feb 5th 7:05am"
```

Comment:
@tansaku Is these lines ok because I'm not sure on the logic how does this thing makes `@recent_hangout` not nil? Almost seems like magic. I think I'll be able to better explain this during a hangout when you will return.